src/JsonSchema/docs/JsonSchema.md

```diff
@@ -0,0 +1,3 @@
+# JSON Schema
+
+TBD
```

Comment:
Let's work on this README later ;)

Reply:
Yes, this whole component library needs to be documented.